tests/test_shell_script_generator.py

```diff
@@ -114,7 +121,8 @@ def test_generate_shell_script(self):
         'oauth': '',
         'project_id': 'my-project',
         'storage_location': 'us-east1',
-        'shutdown_timer_in_sec': 500
+        'shutdown_timer_in_sec': 500,
+        'source_image_family_uri': 'projects/my-dataproc-project/global/images/family/debian-10'
```

Comment:
Do you expect the user to use an image family from their own project?

Reply:
Yes. See use case in previous conversation.